tools/test/test262/excludelist.orig.xml

```diff
@@ -1488,402 +1488,11 @@
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Tai_Viet"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Takri"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Tamil"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Tangut"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Telugu"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Thaana"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Thai"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Tibetan"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Tifinagh"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Tirhuta"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Ugaritic"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Vai"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Wancho"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Warang_Citi"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Yi"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Script_Extensions_-_Zanabazar_Square"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Sentence_Terminal"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Soft_Dotted"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Terminal_Punctuation"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Unified_Ideograph"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Uppercase"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/Variation_Selector"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/White_Space"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/XID_Continue"><reason>TODO</reason></test>
-    <test id="built-ins/RegExp/property-escapes/generated/XID_Start"><reason>TODO</reason></test>
+    <test id="built-ins/RegExp/property-escapes/generated/Assigned"><reason>Not Working on 12.0 unicode and 12.1 unicode table</reason></test>
```

Comment:
Does generating `UnicodePatternTables.h` with latest platform(with new version of ICU) fix this issue?